src/import/sst_service.rs

```diff
@@ -675,6 +684,53 @@ impl<E: Engine> ImportSstService<E> {
     }
 }
 
+fn check_local_region_stale(
+    region_id: u64,
+    epoch: &RegionEpoch,
+    local_region_info: Option<RegionInfo>,
+) -> Result<()> {
+    match local_region_info {
+        Some(local_region_info) => {
+            let local_region_epoch = local_region_info.region.region_epoch.unwrap();
+
+            // TODO(lance6717): we should only need to check conf_ver because we require all
```

Comment:
`version` is not about the leader, it increases after performing split or merge. I don't think we only need to check `conf_ver`